src/manage_cyhy_ops/manageoperators.py

```diff
@@ -87,7 +87,7 @@ def _update_cyhy_ops_users(self, username: str, remove: bool = False) -> int:
                 Type="SecureString",
                 Overwrite=True,
             )
-            log_msg = f'Successfully {update_msg} CyHy Operators in region "%s"'
+            log_msg = f'{update_msg} in region "%s"'
```

Comment:
Consistently end all log messages (at least from what I see in this PR) with a period.  I leave it as an exercise for the reader to check the rest of the file. 😉 
```suggestion
            log_msg = f'{update_msg} in region "%s".'
```